apps/entity-service/src/repl.ts

```diff
@@ -88,6 +96,18 @@ bootstrapRepl("Entity Service", AppModule, {
         console.log("Finished updating hashes on I18nItems.");
         console.log(`Updated ${bar.total} I18nItems.`);
       });
+    },
+    cleanAudits: async () => {
+      const deletedAuditCount = await Audit.destroy({
+        where: {
+          [Op.and]: [
+            where(col("old_values"), "=", col("new_values")),
+            { auditableType: { [Op.ne]: FormSubmission.LARAVEL_TYPE } },
+            { createdAt: { [Op.gt]: new Date(2024, 8, 1) } }
```

Comment:
Looks like this needs to be `2024, 9, 1`: https://github.com/wri/wri-terramatch-api/blob/4f128c81c3d33031b98d6231a84265af11f481bb/app/Http/Controllers/V2/AuditStatus/GetAuditStatusController.php#L40-L43